eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -5954,4 +6007,37 @@ public function testCreateContentWithRomanianSpecialCharsInTitle()
         $urlAlias = $urlAliasService->lookup($expectedPath);
         $this->assertSame($expectedPath, $urlAlias->path);
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\Content[] $drafts
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    private function createContentWithReverseRelations(array $drafts)
```

Comment:
What is the return type hint here?

Reply:
I will add `@return object`. Return type hint is available since PHP 7.2+